addons/event/models/event_registration.py

```diff
@@ -48,6 +55,10 @@ class EventRegistration(models.Model):
         ('open', 'Confirmed'), ('done', 'Attended')],
         string='Status', default='draft', readonly=True, copy=False, tracking=True)
 
+    @api.depends('event_id')
+    def _compute_available_event_ticket_ids(self):
+        self.available_event_ticket_ids = self.mapped("event_id.available_event_ticket_ids")
```

Comment:
you should add a loop ? (mapped is not mandatory from V14)
```
for rec in self:
     rec.available_event_ticket_ids = rec.event_id.available_event_ticket_ids
```

Reply:
Your code is wrong, the suggestion of @fmdl is 100% correct.